### ISSUE TITLE
Error when starting app

### ISSUE THREAD

**Issue by benoj** [formio/formio#395]
Hi,

Startingt he application results in `router.post` is not a function.

Have setup mongodb started with options Y -> 1 -> Enter

When loading default template get the following:
```
readline.js:1015
            throw err;
            ^

TypeError: router.post is not a function
    at module.exports (/Users/benflowers/Projects/candidate/formio/src/templates/import.js:460:10)
    at importTemplate (/Users/benflowers/Projects/candidate/formio/install.js:378:55)
    at /Users/benflowers/Projects/candidate/formio/node_modules/async/lib/async.js:718:13
    at iterate (/Users/benflowers/Projects/candidate/formio/node_modules/async/lib/async.js:262:13)
    at /Users/benflowers/Projects/candidate/formio/node_modules/async/lib/async.js:274:29
    at /Users/benflowers/Projects/candidate/formio/node_modules/async/lib/async.js:44:16
    at /Users/benflowers/Projects/candidate/formio/node_modules/async/lib/async.js:723:17
    at /Users/benflowers/Projects/candidate/formio/node_modules/async/lib/async.js:167:37
    at /Users/benflowers/Projects/candidate/formio/install.js:326:9
    at /Users/benflowers/Projects/candidate/formio/node_modules/prompt/lib/prompt.js:316:32
    at /Users/benflowers/Projects/candidate/formio/node_modules/utile/node_modules/async/lib/async.js:142:25
    at assembler (/Users/benflowers/Projects/candidate/formio/node_modules/prompt/lib/prompt.js:313:9)
    at /Users/benflowers/Projects/candidate/formio/node_modules/prompt/lib/prompt.js:322:32
    at /Users/benflowers/Projects/candidate/formio/node_modules/prompt/lib/prompt.js:597:5
    at Interface.onLine (/Users/benflowers/Projects/candidate/formio/node_modules/read/lib/read.js:111:5)
    at emitOne (events.js:115:13)
    at Interface.emit (events.js:210:7)
    at Interface._onLine (readline.js:278:10)
    at Interface._line (readline.js:625:8)
    at Interface._ttyWrite (readline.js:904:14)
    at ReadStream.onkeypress (readline.js:157:10)
    at emitTwo (events.js:125:13)
```

**Comment by benoj** [comment]
node -v : v8.1.2
npm -v: 5.0.3

**Comment by benoj** [comment]
dropping back tag from v1.18.13 -> 1.18.11 seems to resolve the issue.

**Comment by juanitomint** [comment]
Hi there the problem seems to be on the added lines 460 to 477 to file:

/templates/import.js

introduced on v1.18.12 tag by this commit:
```
Version bump and export/import code cleanup.
SHA 7b8dc867
by Travis Tidwell, 08/07/2017 05:35 PM
parent 767c424e

```

Commenting out this lines seems to resolve the problem of install.
I don't know if has other implications over the app.

**Comment by travist** [comment]
I will take a look right now.

**Comment by travist** [comment]
This is now fixed in version 1.18.14. Sorry about the issue and thanks for the help.
